### PR TITLE
Isolate create config from bin crate for reusability

### PR DESCRIPTION
**Pull request by edmondop**
To address #13, we need to have access to helper methods to manage configuration not located in `isograph_schema`

### CODE REVIEW

File: crates/isograph_api/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+isograph_schema = { path = "../isograph_schema" }
+lazy_static = "1.4"
+regex = "1.6.0"
+colored = "2.0.4"
```
Comment (rbalicki2) [inline crates/isograph_api/Cargo.toml:12]: are these used?

File: crates/isograph_api/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[package]
+name = "isograph_api"
```
Comment (rbalicki2) [inline crates/isograph_api/Cargo.toml:2]: can we make this a bit more specific, e.g. `isograph_config`? `api` is kinda vague.

File: crates/isograph_api/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+isograph_schema = { path = "../isograph_schema" }
+lazy_static = "1.4"
+regex = "1.6.0"
+colored = "2.0.4"
+serde = "1.0.190"
+serde_json = "1.0.108"
```
Comment (rbalicki2) [inline crates/isograph_api/Cargo.toml:14]: serde and serde_json are maybe not needed in `isograph_cli` now

**Review by rbalicki2 — CHANGES_REQUESTED** [review]
Love it! Just a few small nits

File: crates/isograph_api/Cargo.toml (hunk 2 above)
Comment (edmondop) [reply crates/isograph_api/Cargo.toml]: Fixed with [`1236591` (#26)](https://github.com/isographlabs/isograph/pull/26/commits/12365914ff55d6c3839f90efabf8498be374ee1a)

File: crates/isograph_api/Cargo.toml (hunk 3 above)
Comment (edmondop) [reply crates/isograph_api/Cargo.toml]: Fixed with [`1236591` (#26)](https://github.com/isographlabs/isograph/pull/26/commits/12365914ff55d6c3839f90efabf8498be374ee1a)

**Pull request merged**